lsp-mode.el

```diff
@@ -6195,9 +6195,11 @@ information, for example if it doesn't support DocumentSymbols."
   "Render INPUT0, an `&DocumentSymbol', to a string.
 If SHOW-DETAIL? is set, make use of its `:detail?' field (often
 the signature)."
-  (let ((base (or (and show-detail? detail?) detail? name)))
-    (if deprecated? (propertize base 'face 'lsp-face-semhl-deprecated)
-      base)))
+  (let ((detail (and show-detail? detail? (not (string-empty-p detail?))
+                     (propertize (concat ": " detail?) 'face 'font-lock-type-face)))
```

Comment:
I would use face like `lsp-lens-face` - height 0.8 and  :inherit shadow

Reply:
But isn't `font-lock-type-face` more appropriate here, given that :detail? is almost always a type?

Reply:
Yes, but:

1. Check how it looks like in vscode, the focus is on the name, not on the detail. Thus they are using a smaller font .
2. We already use this on multiple places in the project(you may refer on how many places we use lsp-lens-face in lsp-treemacs). Using that will lead to consistent look.

Reply:
Ok, without the colon it also still looks good.

Reply:
`lsp-lens-face` also looks cool, but it is a bit too dark I think:
![20201107-103159](https://user-images.githubusercontent.com/69264154/98437679-8cb16380-20e4-11eb-91d5-972a0bb206c8.png)

Reply:
> `lsp-lens-face` also looks cool, but it is a bit too dark I think:

If you use doom-dark+ which is the emacs port of the vscode theme it will look exactly like the vscode counterpart. In general, treemacs controls are port of vscode one.

Reply:
We could also just make the face customizable.